inlong-manager/manager-dao/src/main/resources/mappers/StreamSinkEntityMapper.xml

```diff
@@ -480,5 +480,15 @@
             is_deleted = 0
         </where>
     </select>
+    <select id="selectAllStreams"
+            resultType="org.apache.inlong.manager.common.pojo.sortstandalone.SortSourceStreamInfo">
+        select inlong_cluster_name as sortClusterName,
+               sort_task_name,
+               inlong_group_id as groupId
+        from stream_sink
+        <where>
```

Comment:
If there are no more where sentences, can be simply as:
```xml
where is_deleted = 0
```

At the same time, **this result set may be very large**. Are you sure you need to find all the results at once?

Reply:
It will reload all configs each 10 seconds and cached them in memory, the overhead is acceptable.